### PR TITLE
Modify Quantity documentation

### PR DESCRIPTION
**Pull request by hettlage**
To make it easier for new users I added a note how to easily create Quantity instances (which I copied from [http://docs.astropy.org/en/stable/units/index.html?highlight=quantity#module-astropy.units.quantity](http://docs.astropy.org/en/stable/units/index.html?highlight=quantity#module-astropy.units.quantity)).

### CODE REVIEW

**Comment by bsipocz** [comment]
@hettlage - Thanks. 

Could you please move the example to the bottom of the docstring under a new section? 

```
Examples
--------
```

Also this doesn't need a changelog entry.

**Comment by mhvk** [comment]
While I see the point that it is useful to point out that quantities can be created more easily, I'd prefer not to duplicate documentation. Instead, I think it makes most sense to just refer to the on-line documentation, maybe by
```
Notes
-----

Quantities can also be created by multiplying a number or array with a
`~astropy.unit.Unit`. See http://docs.astropy.org/en/latest/units/
```
This would still be at the bottom; at least my guess would be that most people doing `u.Quantity?` in ipython would just want to know the options.

**Comment by crawfordsm** [comment]
Hi Christian -- Looks like a pep8 problem on line 139 due to white space.

File: astropy/units/quantity.py [hunk 1]
```diff
@@ -136,7 +136,7 @@ def default_format(val):
 @six.add_metaclass(InheritDocstrings)
 class Quantity(np.ndarray):
     """ A `~astropy.units.Quantity` represents a number with some associated unit.
-
+
```
Comment (mhvk) [inline astropy/units/quantity.py]: This extra space is the one referred to by @bsipocz

**Comment by mhvk** [comment]
@hettlage - looks all good except for that space. Do you know how to rebase and squash all your work into one commit? If so, could you do that? If not, I'm happy to do it for you (and I can remove that space as well, all ensuring the commit will remain attributed to you).

**Comment by hettlage** [comment]
@mhvk - Thanks! It would be great if you could do the rebase, as I've never done one before.

**Comment by astrofrog** [comment]
@hettlage - if you like I can walk you through the steps - I'm in the [Astropy Gitter chat room](https://gitter.im/astropy/astropy?utm_source=share-link&utm_medium=link&utm_campaign=share-link) if you want - just mention ``@astrofrog`` there and I'll be notified.

**Comment by mhvk** [comment]
@hettlage - OK, unless I hear otherwise, I'll do the rebase in an hour or so.

**Comment by hettlage** [comment]
@mhvk - Thanks. But @astrofrog has talked me through the rebase process, so the latest version *should* be rebased now.

**Comment by mhvk** [comment]
@hettlage - OK, yes, I should have looked better. Sadly, still one pep8 mistake! Can you fix that too?

1. edit `quantity.py`
2. `git add astropy/units/quantity.py`
3. `git commit --amend`
4. `git push -f`

**Comment by astrofrog** [comment]
Just to be clear, the issue is that there are extra spaces on line 139

File: astropy/units/quantity.py [hunk 2]
```diff
@@ -185,6 +185,12 @@ class Quantity(np.ndarray):
     TypeError
         If the unit provided is not either a :class:`~astropy.units.Unit`
         object or a parseable string unit.
+
+    Notes
+    -----
+    Quantities can also be created by multiplying a number or array with a
+    :class:`~astropy.unit.Unit`. See http://docs.astropy.org/en/latest/units/
```
Comment (astrofrog) [inline astropy/units/quantity.py]: @hettlage - there is an extra space at the end of this line, can you remove it?

File: astropy/units/quantity.py [hunk 3]
```diff
@@ -185,6 +185,12 @@ class Quantity(np.ndarray):
...
```
Comment (mhvk) [inline astropy/units/quantity.py]: The sphinx build fails because this should be `~astro.units.Unit`...

**Comment by mhvk** [comment]
@hettlage - thanks for fixing, and for keeping what must have seemed a bit of a thankless task... Will merge now!

**Pull request merged**